app/assets/stylesheets/_style.css.scss

```diff
@@ -22,18 +21,22 @@ nav {
 
   .inner-left {
     @include span-columns(3 of 4);
-    height: $_nav_height;
-    line-height: $_nav_height;
+    height: $nav_height;
+    line-height: $nav_height;
 
     >div {
       display: inline-block;
     }
+
+    .mobile-control {
+      display: none;
+    }
   }
 
   .inner-right {
     @include span-columns(1 of 4);
-    height: $_nav_height;
-    line-height: $_nav_height;
+    height: $nav_height;
```

Comment:
Name of variable `nav_height` should be written in all lowercase letters with hyphens instead of underscores